static/js/main.js

```diff
@@ -423,8 +423,25 @@ function fetchVersion() {
         });
 }
 
+// Initialize hotkey functionality.
+function initializeHotkeys() {
+    document.addEventListener('keydown', (event) => {
+        if (event.code === 'KeyB') {
+            toggleSelection();
+        }
+        if (event.code === 'KeyF') {
+            captureFrame();
+        }
+        if (event.code === 'KeyS') {
+            save();
+        }
+        // Later add arrow keys for frame step in video.
```

Comment:
I don't like this comment.
We generally want comments to talk about current code, not past or future.
You can use a TODO comment, or just delete it.